prosper/crons/FetchMapInfo.py

```diff
@@ -0,0 +1,100 @@
+'''FetchMapInfo.py: a script for saving the map statistics from EVE Online XML API'''
+
+from os import path #TODO: path->plumbum?
+
+import requests
+import pandas
+import xml.etree.ElementTree as ET
+from plumbum import cli
+
+from prosper.common.prosper_logging import create_logger
+from prosper.common.prosper_config import get_config
+
+HERE = path.abspath(path.dirname(__file__))
+ME = __file__.replace('.py', '')
+CONFIG_ABSPATH = path.join(HERE, 'cron_config.cfg')
+
+config = get_config(CONFIG_ABSPATH)
+logger = create_logger(
```

Comment:
Logger build will need to be fixed to [new version](https://github.com/EVEprosper/ProsperCommon/blob/master/docs/prosper_logging.md)

can provide a counter-pull with logger fixed